uaclient/messages.py

```diff
@@ -1202,6 +1202,19 @@ class TxtColor:
     "Value provided was not found in {enum_class}'s allowed: value: {values}",
 )
 
+LIVEPATCH_KERNEL_UPGRADE_REQUIRED = NamedMessage(
+    name="livepatch-kernel-upgrade-required",
+    msg="""\
+The running kernel has reached the end of its active livepatch window.
+Please upgrade the kernel with apt and reboot for continued livepatch support.""",  # noqa: E501
+)
+LIVEPATCH_KERNEL_EOL = FormattedNamedMessage(
+    name="livepatch-kernel-eol",
+    msg="""\
+The current kernel ({version}, {arch}) has reached the end of its livepatch support.
+Supported kernels are listed here: https://ubuntu.com/security/livepatch/docs/kernels
+Either switch to a supported kernel or `pro disable livepatch` to dismiss this warning.""",  # noqa: E501
+)
```

Comment:
@kian99 Can you confirm that these messages make sense for the "kernel-upgrade-required" and "kernel-end-of-life" support statuses, respectively?

Reply:
These look good to me.